### ISSUE TITLE
Default mouse cursor gets reset from time to time

### ISSUE THREAD

**Issue by mbert** [alexzielenski/Mousecape#43]
I cannot (yet) reproduce this when needed, but I've had this effect several times since I set up Mousecape yesterday: I applied a theme (Swanslös Blue), maybe worth mentioning that I configured the cursor to be slightly larger than the default one. 

Several times after logging in after hibernation, but not sure if only after logging in, I noticed that I had the original default cursor again. Interestingly the "wait" cursor was still the one from the theme. So I had to open Mousecape again and re-apply the theme to have the styled cursor back.

**Comment by windtrader** [comment]
I'm losing Mousecape cursor quite often. It seems related to disconnecting/ reconnecting Macbook Pro from/to power charger, second display, or Ethernet. 

Sometimes it loses all the cursor icons, then sometimes it loses some of them. For example, at this moment the text I bar icon is lost but the link hover icon remains Mousecape.

**Comment by alexzielenski** [comment]
Yeah the problem is that Mousecape is a one-off operation that sets the cursor image data once. The system can then overwrite that at any point and there's no way of me knowing that it's happened without polling the registered cursor data every so often to make sure it is set.

**Comment by windtrader** [comment]
What I do is place Mousescape in the bar and a quick click, CTL enter, CTL Q and the cursor is restored. Minor issue at most, now that the behaviour is understood. thanks

**Comment by 7stud** [comment]
My cursor resets the default when I reboot.

**Comment by alexzielenski** [comment]
And you guys have installed the helper tool? It should reapply once you login the problem here is that later on while logged in for some people it doesn't retain its application.

**Comment by 7stud** [comment]
What is the helper tool?

**Comment by alexzielenski** [comment]
It applies the cursor on logins, display setting changes, and user context switches automatically.

**Comment by windtrader** [comment]
I could not find the helper tool to install. Kindly provide link or location for installing. Thanks

**Comment by bunam** [comment]
launch mousecap 
in the Mousecpp menu you have Install Helper Tool

**Comment by aaronhyperum** [comment]
With the helper tool installed, El Capitan 10.11.2, reboot changes the cursor back to normal. Is the helper tool not working?

**Comment by fnankivell** [comment]
For me too the helper tool is not working in El Capitan when installed with Mousecape. Simple workaround is to manually add it to your startup items. The helper tool is located here:

Mousecape.app/Contents/Library/LoginItems/com.alexzielenski.mousecloakhelper.app

**Comment by david-am** [comment]
How did you manage to get it working?
Installing Helper Tool didn't work for me either. I'm on 10.11.3 too, and I tried adding com.alexzielenski.mousecloakhelper.app to Login Items (by dragging it, since it can't be added using the + button) but cursors are still being reset to default in every restart/shut down/log off. Is there any step I'm missing? Must SIP be disabled?

**Comment by fnankivell** [comment]
I'm pretty sure SIP must be disabled. Did you give it a try?

**Comment by david-am** [comment]
Yes, I tried disabling SIP but still didn't work.
But I realized that next to "Applied Cape" it was shown "None" instead of the name of the custom cape applied, even though the cursors changed to the custom ones, and that may actually be the problem.
Instead of creating a new cape, I was using a cape obtained by dumping system cursors, modifying some of the cursors within the cape and applying it.
When applying said cape cursors changed to the custom ones, but only till the next restart/shutdown, and "None" was shown next to "Applied Cape". Then I deleted the cape I obtained by dumping cursors, created a new one from zero and applied it. This time, the cape name was shown next to "Applied Cape" right after applying the cape, and the cape was automatically applied after every restart/shutdown.
So far Mousecape is working fine for me, running OS X 10.11.3, without needing to disable SIP nor even adding com.alexzielenski.mousecloakhelper.app to Login Items. I just installed the Helper Tool, created the custom cape from zero and it worked!

**Comment by DMZPvP** [comment]
My cursor keeps showing up and the when i close mousecape it just disappears how do i fix this and please put the link for the helper tool

**Comment by NoobDestr0y3r** [comment]
i deleted mousecape without turning the cap of can any1 help me?

**Comment by NordJan** [comment]
Yes, re-install MouseCape and with the app running, restore defaults (Cmd+R) and re-delete Mousecape .

**Comment by NoobDestr0y3r** [comment]
thx
